.github/opencollective.yml

```diff
@@ -0,0 +1,77 @@
+# Collective
+# ----------
+
+# Define the Open Collective you want to link the repository with.
+# Please use the Open Collective slug not the Open Collective URL.
+
+collective: celery
+
+# Examples
+
+# collective: webpack
```

Comment:
webpack?

Reply:
this is an example. we should remove the comments.